README.md

```diff
@@ -40,32 +40,32 @@
  On this page display the names of files & folders (as determined by an API call) for the selected usernames’ repository.
  
  ## Task 7.
-  Stretch exercise.
+  //Stretch exercise.
   
-  Make a new route GET /github/users/:username/repos/:repoName/*path
+  //Make a new route GET /github/users/:username/repos/:repoName/*path
   The path will be path to file I.E app/controllers/HelloWorldController.scala, you can base64 encode path for ease
   If the path given resolves to a file, display the plaintext of the file on the page.
   If the path given is a folder, display the files & folder names (In the same way it's displayed in the first part of Task 4)
   
 ## Task 8. 
- (Assuming task 7 stretch is complete)
+ //(Assuming task 7 stretch is complete)
  Link up the pages. I.E the first step of task 5 doesn't only list the files/folders, but they are hyperlinks that will direct to the correct page in the  stretch exercise.
  For example, /github/users/jxr227/repos/ROScratch, will show various files and folders including the folder "exercises". If you click "exercises" you  will be directed to:
  /github/users/jxr227/repos/ROScratch/exercises
  
  Do the same on the /*path pages, so that you can, from the top level of a repository, browse all folders and files.
 
 ## Task 9.
- if you haven't already done so write integration and unit tests for all your methods. When it comes to unit testing your connector (the file using ws client that connects to the API) you can use wiremock to mock the API responses. 
+ //if you haven't already done so write integration and unit tests for all your methods. When it comes to unit testing your connector (the file using ws client that connects to the API) you can use wiremock to mock the API responses. 
```

Comment:
This probably needs to come earlier to help stress test driven development? Up until now they have no way of knowing if what they have done is correct...